src/patternfly/components/Skeleton/skeleton.scss

```diff
@@ -0,0 +1,200 @@
+$pf-c-skeleton--spacer-map: build-spacer-map("xs", "sm", "md", "lg", "xl", "2xl");
+
+.pf-c-skeleton {
+  // Padding modifier
+  --pf-c-skeleton--m-padding--Padding: var(--pf-global--spacer--lg);
+
+  // Container
+  --pf-c-skeleton__container--GridGap: 0;
+  --pf-c-skeleton__container--PaddingTop: 0;
+  --pf-c-skeleton__container--PaddingBottom: 0;
+
+  // Element
+  --pf-c-skeleton__element--BackgroundColor: var(--pf-global--palette--black-150);
+  --pf-c-skeleton__element--Height: var(--pf-global--spacer--lg);
+  --pf-c-skeleton__element--Width: 100%;
+  --pf-c-skeleton__element--BorderRadius: #{pf-size-prem(4px)}; // add variable ?
```

Comment:
Seems like we should be able to use one of our existing border-radius vars?

Reply:
@doruskova would you be ok with using a 3px border radius to align with the system?

Reply:
@christiemolloy sure, no problem. I tried it in my codepen and the difference is minimal. Keep consistency with the system :)